save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/usersettings/right/Profile.kt

```diff
@@ -47,6 +45,7 @@ val profileSettingsCard: FC<SettingsProps> = FC { props ->
     // === states ===
     val avatarWindowOpen = useWindowOpenness()
     val (settingsInputFields, setSettingsInputFields) = useState(SettingsInputFields())
+    val (selectedAvatar, setSelectedAvatar) = useState<String>()
```

Comment:
in this case, `selectedAvatar` is either `String` or `undefined`. It might be more clear and Kotlin-style to use `useState<String?>(null)` (if I am not mistaken)

Reply:
yes, this one is good - will set `null` by for now, anyway it is not used yet